src/Task_21/PersonService.java

```diff
@@ -0,0 +1,74 @@
+package Task_21;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.util.Iterator;
+import java.util.Set;
+import java.util.TreeSet;
+
+public class PersonService {
+
+    public static Set<Person> people = new TreeSet<>(new Comparator());
```

Comment:
Also, using of Set is not correct here, as we could have 2 identical people, which are actually different people